x/nexus/keeper/wasm_message_route.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/axelarnetwork/axelar-core/x/nexus/exported"
+	types "github.com/axelarnetwork/axelar-core/x/nexus/types"
+)
+
+type req struct {
+	RouteMessages []exported.WasmMessage `json:"route_messages"`
+}
+
+// NewMessageRoute creates a new message route
+func NewMessageRoute(nexus types.Nexus, account types.AccountKeeper, wasm types.WasmKeeper) exported.MessageRoute {
+	return func(ctx sdk.Context, _ exported.RoutingContext, msg exported.GeneralMessage) error {
+		if msg.Asset != nil {
+			return fmt.Errorf("asset is not supported")
```

Comment:
```suggestion
			return fmt.Errorf("asset is not nil")
```
or something to signal that no assets are supported

Reply:
"nil" is better than "supported" to "signal that no assets are supported"?

Reply:
I feel like saying "asset is not supported" by itself prompts the question of which assets are supported. Which there are none, so it feels strange. I feel "asset transfers to wasm are not supported" or some such would be clearer.

Reply:
I'll improve this one, not exactly the way you proposed though.